src/sentry/static/sentry/app/views/organizationEvents/utils/getParams.jsx

```diff
@@ -0,0 +1,35 @@
+import {DEFAULT_STATS_PERIOD} from 'app/constants';
+import {defined} from 'app/utils';
+
+// Filters out params with null values and returns a default
+// `statsPeriod` when necessary.
+//
+// Accepts `period` and `statsPeriod` but will only return `statsPeriod`
+//
+// TODO(billy): Make period parameter name consistent
+export function getParams(params = {}) {
```

Comment:
At a glance this looks exactly the same as the original `getParams` function, why is it being copied to a new file?

Reply:
Health is going to be removed.